tracer/test/Datadog.Trace.TestHelpers/SkipOn.cs

```diff
@@ -0,0 +1,74 @@
+// <copyright file="SkipOn.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Datadog.Trace.TestHelpers;
+
+/// <summary>
+/// SkipOn helper
+/// </summary>
+public static class SkipOn
+{
+    /// <summary>
+    /// Platform enum
+    /// </summary>
+    public enum Platform
```

Comment:
Is this necessary? Can't we use the built in `System.Runtime.InteropServices.OSPlatform`?

Reply:
unless I use it inside a `#if NETCOREAPP` I'm forced to use a custom enum and map it later.